sdk/metric/doc.go

```diff
@@ -39,6 +39,30 @@
 // Meter.RegisterCallback and Registration.Unregister to add and remove
 // callbacks without leaking memory.
 //
+// # Cardinality Limits
+//
+// Cardinality refers to the number of unique attribute combinations for a single
+// metric within one collection (export) interval. High cardinality can lead to
```

Comment:
```suggestion
// metric within one collection (export). High cardinality can lead to
```

Reply:
There could be also "manual" (non-interval based) collection e.g. Prometheus exporter. Right?